rules/import-index.js

```diff
@@ -16,19 +16,37 @@ const importIndex = (context, node, argument) => {
 };
 
 const create = context => {
-	return {
-		'CallExpression[callee.name="require"]': node => importIndex(context, node, node.arguments[0]),
-		ImportDeclaration: node => importIndex(context, node, node.source)
+	const options = context.options[0] || {};
+
+	const rules = {
+		'CallExpression[callee.name="require"]': node => importIndex(context, node, node.arguments[0])
 	};
+
+	if (!options.ignoreImports) {
+		rules.ImportDeclaration = node => importIndex(context, node, node.source);
+	}
+
+	return rules;
 };
 
+const schema = [{
+	type: 'object',
+	properties: {
+		ignoreImports: {
+			type: 'boolean'
+		}
```

Comment:
not familiar with this but is default value needed?

Reply:
done